docs/managing-dragonfly/flags.md

```diff
@@ -0,0 +1,659 @@
+
+  default:
+
+### --fromenv
+  Comma-separated list of flags to set from the environment [use 'export FLAGS_flag1=value']). 
+
+  default:
+
+### --tryfromenv
+  Comma-separated list of flags to try to set from the environment if present.
+
+  default: 
+
+### --undefok
+  Comma-separated list of flag names that it is okay to specify on the command line even 
+  if the program does not define a flag with that name.
+
+  default: 
+
+### --alsologtoemail
+  Log messages go to these email addresses in addition to logfiles.
+
+  default: ""
+
+### --alsologtostderr
+  Log messages go to stderr in addition to logfiles.
+
+  default: false
+
+### --colorlogtostderr
+  Color messages logged to stderr (if supported by terminal)). 
+
+  default: false
+
+### --colorlogtostdout
+  Color messages logged to stdout (if supported by terminal). 
+
+  default: false
+
+### --drop_log_memory
+  Drop in-memory buffers of log contents. Logs can grow very quickly and they are rarely read 
+  before they need to be evicted from memory. Instead, drop them from memory as soon as they 
+  are flushed to disk. 
+
+  default: true
+
+### --log_backtrace_at
+  Emit a backtrace when logging at file:linenum.
+
+  default: ""
+
+### --log_dir
+  If specified, logfiles are written into this directory instead of the default logging directory.
+
+  default: ""
+
+### --log_link 
+  Put additional links to the log files in this directory. 
+
+  default: ""
+
+### --log_prefix 
+  Prepend the log prefix to the start of each log line.
+
+  default: true
+
+### --log_utc_time 
+  Use UTC time for logging.
+
+  default: false
+
+### --log_year_in_prefix 
+  Include the year in the log prefix. 
+
+  default: true
+
+### --logbuflevel 
+  Buffer log messages logged at this level or lower (-1 means don't buffer; 0 means buffer INFO only; ...)
+
+  default: 0
+
+### --logbufsecs
+  Buffer log messages for at most this many seconds.
+
+  default: 30
+
+### --logcleansecs
+  Clean overdue logs every this many seconds).
+
+  default: 300
+
+### --logemaillevel 
+  Email log messages logged at this level or higher (0 means email all; 3 means email FATAL only; ...). 
+
+  default: 999
+
+### --logfile_mode 
+  Log file mode/permissions.
+
+  default: 436
+
+### --logmailer
+  Mailer used to send logging email.
+
+  default: ""
+
+### --logtostderr
+  Log messages go to stderr instead of logfiles.
+
+  default: false
+
+### --logtostdout 
+  Log messages go to stdout instead of logfiles.
+
+  default: false
+
+### --max_log_size
+  Approx. maximum log file size (in MB). A value of 0 will be silently overridden to 1. 
+
+  default: 1800
+
+### --minloglevel 
+  Messages logged at a lower level than this don't actually get logged anywhere. 
+
+  default: 0
+
+### --stderrthreshold 
+  Log messages at or above this level are copied to stderr in addition to logfiles. This flag obsoletes --alsologtostderr;
+  
+  default: 2
+
+### --stop_logging_if_full_disk
```

Comment:
I would remove "secondary flags" that come from thirdparty libraries besides few whitelised ones like logtostderr/out.